streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -272,7 +287,6 @@ private static void flush(final KafkaProducer<byte[], byte[]> producer,
             producer.send(new ProducerRecord<>(
                 partition.topic(),
                 partition.partition(),
-                System.currentTimeMillis() + Duration.ofDays(2).toMillis(),
```

Comment:
Could you put `System.currentTimeMillis()` so that I do not need to navigate to the method to understand that the timestamp is set to `System.currentTimeMillis()` by default? 🙂

Reply:
Done